### ISSUE TITLE
Genital Preference

### ISSUE THREAD

**Issue by JapanYoshi** [Goctionni/KinkList#7]
The list lacks a “penis”, “vagina”, and possibly “breasts” options in addition to the body type options. I think it would be a gaping hole to leave this out, since for many people this would be separate from the gender identity of the partner.

In addition, “futanari” should be split into “hermaphrodite” (original meaning, meaning penis + vagina in one body), and another option meaning penit + breasts in one body, but not necessarily a vagina.

**Comment by Karadoll** [comment]
The term futanari is only for fantasy style hermaphrodites, sure in certain circles it's used to mean the later as well but that is a misuse and that is transfemale.

**Comment by JapanYoshi** [comment]
That doesn't really address the question though.

**Comment by Karadoll** [comment]
Yes it does, as in it's not an issue or something that should be changed as what you are proposing is an incorrect use of the terminology.

**Comment by JapanYoshi** [comment]
Did you even read the rest?

**Comment by Goctionni** [comment]
I will consider both sides of this discussion / argument in the next release if/when it comes. I understand how specifying a partner's preferred genitals may be simpler, though I think it can also cause some confusion if your preferences are different for different persons. IE: someone may be attracted to male, to female, but not to characters having both genital sets. I will consider the subject when it comes to a next release.
